modules/collect/src/main/java/com/opengamma/strata/collect/Guavate.java

```diff
@@ -967,7 +1004,7 @@ public static ThreadFactoryBuilder namedThreadFactory(String threadNamePrefix) {
    * <p>
    * This takes an argument which is the number of stack levels to look back.
    * This will be 2 to return the caller of this method, 3 to return the caller of the caller, and so on.
-   * 
+   *
```

Comment:
Really need an IntelliJ setting to avoid this

Reply:
It's frustrating - I tried to add the space and commit it back, but intellij refused to see it as a  change it could commit!